rpc/openapi/openapi.yaml

```diff
@@ -537,8 +537,7 @@ paths:
             type: array
```

Comment:
Maybe this is fine, but are these changes actually in the v0.34 line? This didn't seem to be related to the CI parts of the change offhand.

Reply:
These fixes are just to get the linter to pass. I also went and checked the response schemas against the structs on the `main` branch and it seems there were a few minor inconsistencies there, so I fixed the schema where it was incorrect.